lib/.coveragerc

```diff
@@ -1,4 +1,11 @@
 [run]
 omit =
-    # Ignore python protobufs
-    streamlit/proto/*_pb2.py
+    streamlit/proto/*
+    streamlit/hello/*
+    streamlit/static/*
+
+[report]
+omit =
+    streamlit/proto/*
+    streamlit/hello/*
+    streamlit/static/*
```

Comment:
According to the [docs here](https://coverage.readthedocs.io/en/7.6.1/excluding.html#advanced-exclusion), we should be able to add a block here to exclude `TYPE_CHECKING` blocks from the coverage report:

```toml
exclude_also =
    if TYPE_CHECKING:
```

Reply:
Nice :) I excluded it 👍